.env.example

```
@@ -71,3 +71,6 @@ REACT_APP_GENERAL_MAINTENANCE_MESSAGE="Message to display in top banner."
 
 # Publish to Storybook manually by adding the frontend-v2 chromatic project id
 CHROMATIC_PROJECT_TOKEN = 
+
+# Disable bridge send transactions in UI and show an error when this is set to truthy (e.g: 1, `true`, etc)
+REACT_APP_BRIDGE_DISABLED = 
```

We should also change this for CHROMATIC_PROJECT_TOKEN.
```suggestion
REACT_APP_BRIDGE_DISABLED= 
```